CHANGES.md

```diff
@@ -9199,6 +9199,7 @@ OpenSSL 1.0.1
            EC_GFp_nistp224_method()
            EC_GFp_nistp256_method()
            EC_GFp_nistp521_method()
+           EC_GFp_secp256k1_method()
```

Comment:
This need a new entry, like was done for p384. This section is related to the 1.0.1 version of OpenSSL.